src/mosquito/base.cr

```diff
@@ -46,5 +49,13 @@ module Mosquito
     def self.log(*messages)
       logger.log(Logger::Severity::INFO, messages.join(" "))
     end
+
+    def self.idle_wait=(idle_wait : Int)
```

Comment:
This is a handy overload, but why not loosen the type constraints on idle_wait a bit so it can be used for other numbers which can have `to_f64` called on them?